packages/wallet/frontend/src/components/userCards/UserCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { CopyButton } from '@/ui/CopyButton'
+import { NormalCard } from '../icons/NormalCard'
+import { DetailsCard } from '../icons/DetailsCard'
+import Image from 'next/image'
+import { CardTypes } from '@/lib/context/card'
+
+type UserCardProps = {
+  type: CardTypes
+}
+
+export const UserCard = ({ type }: UserCardProps) => {
+  return (
+    <div className="w-full h-52 items-center flex justify-center">
+      {type === 'normal' ? (
+        <>
+          <NormalCard className="-z-10 absolute" />
+          <span className="text-white uppercase relative top-[73px] -left-[91px]">
+            Timi Swift
+          </span>
+        </>
+      ) : type === 'details' ? (
+        <>
+          <DetailsCard className="-z-10 absolute" />
+          <div className="text-white uppercase relative top-[22px] left-[16px]">
+            <span>1234 5678 9012 3456</span>
+            <CopyButton
+              aria-label="copy key id"
+              className="h-5 w-5"
+              copyType="card"
+              value="1234 5678 9012 3456"
+            />
+          </div>
+          <span className="text-white uppercase relative top-[77px] -left-[194px]">
+            09/30
+          </span>
+          <span className="text-white uppercase relative top-[74px] -left-[174px]">
+            999
+            <CopyButton
+              aria-label="copy key id"
+              className="h-5 w-5"
+              copyType="card"
+              value="999"
+            />
+          </span>
+        </>
+      ) : (
+        <>
+          <NormalCard className="-z-10 absolute" />
+          <span className="text-white uppercase relative top-[73px] left-[24px] opacity-40">
+            Timi Swift
+          </span>
+          <Image
```

Comment:
for frozen card I tried to copy as svg card from figma with the frozen overlay....but it was such a large file, that it gave out of bounds error. So I added it as an image on top of a normal card view.

Reply:
Let's add a TODO: Maybe we can build the card with CSS only instead of having an SVG for it. For the frozen part, this is okay for now.